### ISSUE TITLE
Split Button out from CreditCard component

### ISSUE THREAD

**Issue by llampwall** [weareseeed/react-square-web-payments-sdk#67]
### Describe the bug

Currently the CreditCard field includes the submit payment button in it. There is no way to separate them. This makes using the library impossible for someone like me who has the button on a different part of the screen, in a different component. When using the regular Square JavaScript SDK, you can put them wherever you want in relation to each other.

`<div id="card-container"></div>
 <button id="card-button" type="button">Pay $1.00</button>`

This library should be updated to split those components and allow them to each be placed anywhere within the children of PaymentForm.

### Your Example Website or App

this applies to any site this is used on

### Steps to Reproduce the Bug or Issue

1. Attempt to put the button somewhere else in your code.
2. Realize there is no trivial way to do so.

### Expected behavior

As a user, I expected to be able to have the same flexibility as the Square Web Payments SDK this library wraps. This is a breaking change.

### Platform

- OS: macOS, Windows
- Browser: Chrome, Safari, Firefox

**Comment by KalebMills** [comment]
Agreed with this. Even though our application houses the two together, we have essentially no control over it.

**Comment by FeLiNa22** [comment]
I agree as well. Also can you make the CreditCard button interoperable with libraries like MUI. Atm it will throw many errors and bugs when trying to compose this component.

**Comment by Sexual** [comment]
Any updates on this? Much needed improvement as it completely destroys our current billing flow.

**Comment by alexbrazier** [comment]
I'm also interested in this option, an example use case would be to create a checkout page like this
<img width="521" alt="image" src="https://github.com/weareseeed/react-square-web-payments-sdk/assets/5689874/76209c09-ba64-431a-805a-a3e501d85b28">

**Comment by 100tomtomas100** [comment]
Why did the merge died  ?
